Fitness/Controllers/HabitTrackingController.swift

```diff
@@ -453,10 +468,12 @@ extension HabitTrackingController: UITableViewDelegate, UITableViewDataSource {
                 cell.setTitle(activity: habit)
                 cell.swipeRight()
                 cell.delegate = self
+                cell.separator.isHidden = true
                 
                 return cell
             } else {
                 // Empty state featured habit
+                // TODO - replace this with the NoHabitsCell class once it is converted to a tableview
```

Comment:
Does this TODO still need to be done?

Reply:
Not now--In the refactor period I'd like to convert the home page from a collectionView to a tableView. Once that's done we can reuse the cell from the home page, but for now I didn't think it made sense to have two versions of the same cell